app/views/merchants/show.html.erb

```diff
@@ -1,4 +1,10 @@
-<h1><%= link_to @merchant.name, "/merchants/#{@merchant.id}" %></h1>
+<% if current_user %>
+  <% if current_user.admin? %>
+    <h1><%= link_to @merchant.name, "admin/merchants/#{@merchant.id}" %></h1>
```

Comment:
This is not linking to the link that is posted. I think it has to do with namespacing some of the nested admin routes but not all of them. Hoping that @Gallup93 and I can transform the other admin routes and then maybe this will magically work.